controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -234,10 +234,17 @@ function execute($request) {
 
 			$user = $request->getUser();
 
+			// Get a list of author user IDs
+			$authorUserIds = array();
+			$stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO');
+			$submitterAssignments = $stageAssignmentDao->getBySubmissionAndRoleId($submission->getId(), ROLE_ID_AUTHOR);
+			$submitterAssignment = $submitterAssignments->next();
```

Comment:
This is a temporary fix that'll need attention when we deal with payments. Personally I suspect we'll have to tear out all the current payment stuff and start from scratch to get it done cleanly. We do have principal contact, but only for author records, not user records, so we can't use that. I'd rather that we had correspondence go to all assigned authors, and if we find we need a principal contact, then we'll have to add a flag to the stage assignment for that.